samples/net/sockets/tls_configurations/prj.conf

```diff
@@ -0,0 +1,55 @@
+# Kernel options
+CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_INIT_STACKS=y
+CONFIG_POSIX_API=y
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+CONFIG_HEAP_MEM_POOL_SIZE=2048
+
+# Generic networking options
+CONFIG_NETWORKING=y
+CONFIG_NET_MAX_CONTEXTS=10
+
+# Socket
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POLL_MAX=5
+CONFIG_NET_SOCKETS_SOCKOPT_TLS=y
+CONFIG_NET_SOCKETS_TLS_MAX_CONTEXTS=4
+
+# Disable native ethernet driver (using native sockets instead)
+CONFIG_ETH_NATIVE_POSIX=n
+
+# Use native sockets
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
+CONFIG_NET_DRIVERS=y
+CONFIG_NET_NATIVE_OFFLOADED_SOCKETS=y
+
+# TLS configuration
+CONFIG_MBEDTLS=y
+CONFIG_MBEDTLS_BUILTIN=y
+CONFIG_MBEDTLS_ENABLE_HEAP=y
+CONFIG_MBEDTLS_HEAP_SIZE=60000
+CONFIG_MBEDTLS_SSL_MAX_CONTENT_LEN=2048
+# Build the PSA crypto core so that TLS stack will make use of PSA crypto APIs.
+CONFIG_MBEDTLS_PSA_CRYPTO_C=y
+CONFIG_MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG=y
+CONFIG_ENTROPY_GENERATOR=y
+# These are enabled by default, but we don't want them here. They will be
+# re-enabled by the proper overlay configuration file if required.
+CONFIG_MBEDTLS_KEY_EXCHANGE_RSA_ENABLED=n
+CONFIG_MBEDTLS_TLS_VERSION_1_2=n
+CONFIG_PSA_WANT_KEY_TYPE_AES=n
+CONFIG_PSA_WANT_ALG_CBC_NO_PADDING=n
+
+# Logging
+CONFIG_LOG=y
+CONFIG_PRINTK=y
+
+# Debug log options (optional)
+# CONFIG_NET_LOG=y
+# CONFIG_NET_STATISTICS=y
+# CONFIG_MBEDTLS_LOG_LEVEL_DBG=y
+# CONFIG_MBEDTLS_DEBUG=y
+
+# Disable optimizations
```

Comment:
Why?

Reply:
Leftover from debug sessions. I can comment it out as for the debug CONFIGs or remove it (in case you prefer to have also the debug CONFIGs removed - it depends on the answer to the previous comment).